cmd/configure.go

```diff
@@ -493,21 +494,18 @@ func configurePublishing(ctx context.Context, flags ConfigureGithubFlags) error
 	}
 
 	secrets := make(map[string]string)
-	var publishPaths, generationWorkflowFilePaths []string
+	workflowPaths := make(map[string]targetWorkflowPaths)
 
 	for _, name := range chosenTargets {
```

Comment:
:warning: the `chosenTargets` variable is currently confusing. When multiple targets are set up in the same repository:
<img width="1126" height="176" alt="screenshot_2025-10-31_13:02:44" src="https://github.com/user-attachments/assets/ac0d74e6-e6a6-4710-9400-c0431603b264" />

then selecting one or the other makes no difference since both workflow files will get updated: 
<img width="1516" height="253" alt="screenshot_2025-10-31_13:03:48" src="https://github.com/user-attachments/assets/a7457b15-3fab-4565-9ed6-e3833d4fb04e" />